test/unit/rules/no-dynamic-subexpression-invocations-test.js

```diff
@@ -13,6 +13,18 @@ generateRuleTests({
     '{{something here="goes"}}',
     '<button onclick={{fn something "here"}}></button>',
     '{{@thing "somearg"}}',
+    '<Foo @bar="asdf" />',
```

Comment:
Does any of this need to go into the docs for this rule?

Reply:
I don't think so (these aren't actually related to what the rule is trying to check), it is just to ensure we don't regress again...